draft-ietf-dnsop-svcb-https.md

```diff
@@ -154,8 +154,8 @@ Additional goals specific to HTTPS RRs and the HTTPS use-case include:
 * Support non-default TCP and UDP ports
 * Enable SRV-like benefits (e.g. apex delegation, as mentioned above) for HTTP(S),
   where SRV {{?SRV=RFC2782}} has not been widely adopted
-* Provide an HSTS-like indication signaling
-  for the duration of the DNS RR TTL that the HTTPS scheme should
+* Provide an HSTS-like indication {{!HSTS=RFC6797}} signaling
+  that the HTTPS scheme should
```

Comment:
```suggestion
* Provide a HSTS-like (but DNS TTL limited) indication {{!HSTS=RFC6797}} signaling
  that the HTTPS scheme should
```
Another wording might be to call it "temporary"?  Or is the real difference that HSTS lifetimes may be much longer than typical DNS TTLs?